### PR TITLE
Anchor id support for all sections

### PR DESCRIPTION
**Pull request by mahfouzhseineweev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+### 1.0.20
+#### Additional Feature
+
+- Adding anchor id for all section types (sectionName-sectionId)
+- Showing anchor id under the title of each section in edit mode
```
Comment (jfayad) [inline CHANGELOG.md:5]: kindly update the readme to how it is now

File: src/base/index.vue [hunk 2]
```diff
@@ -1340,6 +1346,12 @@ export default {
         "Your section has been removed, save your page to display this change to your visitors"
       );
     },
+    copyAnchor(anchor) {
+      // let copyText = document.getElementById("myInput");
```
Comment (jfayad) [inline src/base/index.vue:1350]: remove unnecessary comments please

File: src/base/index.vue [hunk 3]
```diff
@@ -1340,6 +1346,12 @@ export default {
...
+      // copyText.select();
+      // copyText.setSelectionRange(0, 99999);
+      navigator.clipboard.writeText(anchor);
```
Comment (jfayad) [inline src/base/index.vue:1353]: given this is just one function, couldn't you simply call it in the @click event ?

**Review by jfayad — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mahfouzhseineweev) [reply CHANGELOG.md:5]: @jfayad I didn't get you on this one, your comment is on CHANGELOG file but it is about README file. Also I have pulled from master so the files has the last changes already

Comment (jfayad) [reply CHANGELOG.md:5]: typo, the changelog is outdated

**Review by jfayad — APPROVED** [review]

**Pull request merged**